public/static/locales/en/manageProjects.json

```diff
@@ -123,5 +131,52 @@
   "longitudeRequired": "Longitude is required",
   "manageProject": "Manage Projects",
   "descriptionForManageProjects": "Showcase your restoration work for monitoring and to receive donations.",
-  "manageProjects": "Manage Projects"
+  "manageProjects": "Manage Projects",
+  "addConservationProjects": "Add Conservation Projects",
+  "areaProtected": "Area Protected (In HA)",
+  "ecosystems": "Ecosystems",
+  "protectionStartedIn": "Protection Started in",
+  "actions": "Actions",
+  "activitySeasons": "Activity Seasons",
+  "ForestProtectionType": "Forest Protection Type/Measures",
+  "ecologicalBenefits": "Ecological Benefits",
+  "socialBenefits": "Social Benefits",
+  "coBenefits": "Co-Benefits",
+  "benefits": "Benefits",
+  "tropicalMoistForest": "Tropical Moist Forest",
+  "dry": "Dry",
+  "ecosystemType": "Please Select Ecosystem Type",
+  "tropicalDryForests": "Tropical Dry Forests",
+  "tropicalConiferousForests":"Tropical Coniferous Forests",
+  "tropicalGrasslandsForests": "Tropical Grasslands Forests",
+  "temperateBroadleafForests": "Temperate Broadleaf Forests",
+  "temperate-coniferous-forests": "Temperate Coniferous Forests",
+  "temperateGrasslandsForests": "Temperate Grasslands Forests",
+  "mediterraneanForests": "Mediterranean Forests",
+  "mangroves": "Mangroves",
+  "deserts": "Deserts",
+  "floodedGrasslands": "Flooded Grasslands",
+  "montaneGrasslands": "Montane Grassland",
+  "borealForests": "Boreal Forests",
+  "tundra": "Tundra",
+  "temperateConiferousForests": "Temperate Coniferous Forests",
+  "tenure": "Tenure",
+  "rent": "Rent",
+  "ownershipType": "Ownership Type",
+  "landOwnershipType": "Land Ownership Type",
+  "restorationProject": "Restoration Project",
+  "addProjetDescription": "You're few steps away from showcasing your work on Plant-for-the-planet platform.",
+  "addProjetContact": "If you need any help contact us at ",
+  "areaProtectedInfo": "Add Some Information",
+  "supportLink": "support@plant-for-the-planet.org",
+  "wrongCoordinates": "Wrong Coordinates",
+  "actionInfo": "The actions your organization is taking to conserve the ecosystem.",
+  "mainChallengeInfo": "The biggest challenge this project is facing.",
+  "motivationInfo": "The motivation behind protecting this specific ecosystem.",
+  "hector": "hector",
+  "areaProtectedValidation": "Please enter Area Protected",
+  "employeeCountValidation": "Please enter Employee Count",
```

Comment:
these are not necessary as you can make these dynamic. {fieldname} is not valid

Reply:
okay